extensions/wikia/Forum/ForumSpecialController.class.php

```diff
@@ -128,7 +137,7 @@ public function createNewBoardModal() {
 					'isRequired' => true,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "isRequired" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)